aws_lambda_powertools/utilities/parameters/__init__.py

```diff
@@ -4,19 +4,22 @@
 Parameter retrieval and caching utility
 """
 
+from .appconf import AppConfigProvider, get_configuration
 from .base import BaseProvider
 from .dynamodb import DynamoDBProvider
 from .exceptions import GetParameterError, TransformParameterError
 from .secrets import SecretsProvider, get_secret
 from .ssm import SSMProvider, get_parameter, get_parameters
 
 __all__ = [
+    "AppConfigProvider",
     "BaseProvider",
     "GetParameterError",
     "DynamoDBProvider",
     "SecretsProvider",
     "SSMProvider",
     "TransformParameterError",
+    "get_configuration",
```

Comment:
I'm wondering if this could cause confusion. Should this be `get_app_config`?

Reply:
sure, good idea.